src/Razor/Microsoft.AspNetCore.Mvc.Razor.Extensions/test/IntegrationTests/CodeGenerationIntegrationTest.cs

```diff
@@ -525,6 +525,49 @@ public void RazorPageWithNoLeadingPageDirective_Runtime()
             var diagnotics = compiled.CodeDocument.GetCSharpDocument().Diagnostics;
             Assert.Equal("RZ3906", Assert.Single(diagnotics).Id);
         }
+
+        [Fact]
+        public void View_WithCssScope()
```

Comment:
The test is within the razor extensions project because otherwise is painful to write.